all-in-one/Dockerfile

```diff
@@ -87,6 +81,6 @@ ENV PERSISTENCE_FILE_PARENT_PATH="oscal-content" \
 
 EXPOSE 8080
 
-WORKDIR /app/
+WORKDIR /app
 
-ENTRYPOINT ["java", "-jar", "oscal-rest-service-app.jar"]
\ No newline at end of file
+CMD ["oscal-rest-service-app.jar"]
```

Comment:
Okay so this is viable because the upstream image already sets `ENTRYPOINT` during the build? Just want to make sure I have some understanding of this.